connectors/setup.go

```diff
@@ -0,0 +1,33 @@
+package connectors
+
+import (
+	"github.com/avigailbr/book_service/config"
+	"gopkg.in/redis.v5"
+)
+
+var BookConnectors *Connectors
+
+type Connectors struct {
+	ElasticClient *ElasticClient
+	RedisClients  *redis.Client
+}
+
+func NewConnectors() error {
+	BookConnectors = &Connectors{
+	}
+	_, err := BookConnectors.NewElasticClient(config.ElasticUrl, config.IndexName, config.TypeName)
+	if err != nil {
+		return err
+	}
+	_, err = BookConnectors.NewRedisClient(config.RedisAdd, config.RedisDB)
+	if err != nil {
+		return err
+	}
```

Comment:
i would initialise the connectors in the end and use the return values:

```suggestion
	esClient, err := BookConnectors.NewElasticClient(config.ElasticUrl, config.IndexName, config.TypeName)
	if err != nil {
		return err
	}
	redisClient, err = BookConnectors.NewRedisClient(config.RedisAdd, config.RedisDB)
	if err != nil {
		return err
	}
	
	BookConnectors = &Connectors{
	    ElasticClient: esClient,
 	    RedisClients: redisClient,
	}
```
Not critical but looks better IMO